napari_aicssegmentation/model/segmenter_model.py

```diff
@@ -16,3 +16,11 @@ class SegmenterModel:
     selected_channel: Channel = None
     workflows: List[str] = None
     active_workflow: str = None
+
+    def reset(self):
```

Comment:
This is very easy to write a test for - so let's do it right now, especially as the model will likely evolve soon.